apps/console/src/features/identity-providers/components/wizards/iproov/iproov-authentication-provider-create-wizard-content.tsx

```diff
@@ -0,0 +1,391 @@
+    /**
+     * Trigger form submit.
+     * @param submitFunctionCallback - Callback.
+     */
+    triggerSubmission: (submitFunctionCallback: () => void) => void;
+    /**
+     * Trigger previous page.
+     * @param previousFunctionCb - Callback.
+     */
+    triggerPrevious: (previousFunctionCb: () => void) => void;
+    /**
+     * Callback to change the wizard page,
+     * @param pageNo - Page Number.
+     */
+    changePageNumber: (pageNo: number) => void;
+    /**
+     * IDP template.
+     */
+    template: IdentityProviderTemplateInterface;
+    /**
+     * Total wizard page count.
+     * @param pageCount - Page number.
+     */
+    setTotalPage: (pageCount: number) => void;
+    /**
+     * Callback to be triggered for form submit.
+     * @param values - Form values.
+     */
+    onSubmit: (values: IproovAuthenticationProviderCreateWizardFormValuesInterface) => void;
+}
+
+const FORM_ID: string = "iproov-authenticator-wizard-form";
+
+/**
+ * Iproov Authentication Provider Create Wizard content component.
+ *
+ * @param props - Props injected to the component.
+ * @returns Functional component.
+ */
+export const IproovAuthenticationProviderCreateWizardContent: FunctionComponent<
+    IproovAuthenticationProviderCreateWizardContentPropsInterface
+> = (
+    props: IproovAuthenticationProviderCreateWizardContentPropsInterface
+): ReactElement => {
+
+    const {
+        triggerSubmission,
+        triggerPrevious,
+        changePageNumber,
+        template,
+        setTotalPage,
+        onSubmit,
+        [ "data-componentid" ]: componentId
+    } = props;
+
+    const { t } = useTranslation();
+
+    const [ idpList, setIdPList ] = useState<IdentityProviderListResponseInterface>({});
+    const [ isIdPListRequestLoading, setIdPListRequestLoading ] = useState<boolean>(undefined);
+
+    /**
+     * Loads the identity provider authenticators on initial component load.
+     */
+    useEffect(() => {
+
+        getIDPlist();
+    }, []);
+
+    /**
+     * Get Idp List.
+     */
+    const getIDPlist = (): void => {
+
+        setIdPListRequestLoading(true);
+
+        getIdentityProviderList(null, null, null)
+            .then((response: IdentityProviderListResponseInterface) => {
+                setIdPList(response);
+            })
+            .catch((error: AxiosError) => {
+                handleGetIDPListCallError(error);
+            })
+            .finally(() => {
+                setIdPListRequestLoading(false);
+            });
+    };
+
+    /**
+     * Check whether IDP name is already exist or not.
+     *
+     * @param value - IDP name.
+     * @returns error msg if name is already taken.
+     */
+    const idpNameValidation = (value: string): string => {
+
+        let nameExist: boolean = false;
+
+        if (idpList?.count > 0) {
+            idpList?.identityProviders.map((idp: IdentityProviderTemplateInterface) => {
```

Comment:
We can use `.some` instead of `.map` here so that iterating would immediately halt once a matching idp name is found, improving performance a little bit.